### ISSUE TITLE
Problem Adding alt text to images with VoiceOver

### ISSUE THREAD

**Issue by Changelingmx** [DagAgren/toot#455]
Hello,

I am using Toot! with VoiceOver on iOS. When I attach a photo to a toot, I am unable to add alt text to it. I can get to the text field where the alt text goes and enter the description for the photo, but I am then unable to navigate away from the text field to finish posting. Flicking to the right or left with one finger just results in VoiceOver bonking and saying, "Text field." Dragging my finger along the screen just results in VoiceOver reporting a blank screen.

**Comment by DagAgren** [comment]
I think this falls under https://github.com/DagAgren/toot/issues/364.
